components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -220,6 +232,9 @@ def main(argv):
     s3_compressor_parser.add_argument(
         "--aws-secret-access-key", type=str, default=None, help="AWS secret access key."
     )
+    s3_compressor_parser.add_argument(
+        "--aws-profile", type=str, default=None, help="AWS config profile."
```

Comment:
nit: just to be consistent.
```suggestion
        "--aws-profile", type=str, default=None, help="AWS configuration profile to use for authentication"
```